save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -37,7 +41,8 @@ class ProcessBuilderInternal {
     /**
      * Temporary directory for stderr and stdout (popen can't separate streams, so we do it ourselves)
      */
-    val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / this::class.simpleName!!).also {
+    val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY /
+            (this::class.simpleName!! + "_" + Clock.System.now().toString().substringAfterLast(".")).toPath()).also {
```

Comment:
```suggestion
            (this::class.simpleName!! + "_" + Clock.System.now().epochSeconds).toPath()).also {
```
Maybe, just with timestamp?

Reply:
Done